ftw/sliderblock/tests/test_sliderblock_view.py

```diff
@@ -20,7 +20,7 @@ def setUp(self):
         self.page = create(Builder('sl content page'))
 
         slick_config = json.dumps(
-            {'autoplay': False, 'autoplaySpeed': 1000, 'arrows': False}
+            {'labels': {'pause': 'Pause', 'play': 'Play', 'prev': 'Previous', 'next': 'Next'}, 'arrows': 'false', 'autoplaySpeed': 1000, 'autoplay': 'false'}
```

Comment:
I think this line should be reverted, since this is data which is going to be stored as config on the block, and we've changed the labels to no longer be persistent, right?